jpa/subsystem/pom.xml

```diff
@@ -95,12 +95,24 @@
 
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>wildfly-ejb3</artifactId>
+            <artifactId>wildfly-ejb3-jakarta</artifactId>
```

Comment:
This isn't needed any longer as 'wildfly-ejb3' is now the jakarta.* artifact.

Reply:
Thanks, removed the `-jakarta`